Assets/Scripts/UserInterface/GameUI/LevelUpRewardScreenController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using UnityEngine;
+using UnityEngine.UI; 
+using System.Collections;
+
+public class LevelUpRewardScreenController : UIController {
+
+    public GameObject newCanvas;
+    public Text text;
+
+    public static bool Leveled;    
+
+    void Start()
+    {
+        newCanvas.SetActive(false);
+        //text.text = "";
+        Leveled = false;
+    }
+
+    void Update()
+    {       
+        if (Leveled)
+        {
+            newCanvas.SetActive(true);
+            Leveled = false;
+            toggleScreen();
+        }
+    }
+
+    public void toggleScreen()
+    {
+        StartCoroutine(toggler());               
+    }
+
+    IEnumerator toggler()
+    {
+        text.text = "LEVEL UP!";
+        yield return new WaitForSeconds(5);
```

Comment:
Can you make this a public variable in the class so it's tunable from the Inspector?